docs/docs/writing-documentation-with-docz.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Writing documentation with Docz"
+---
+
+Writing good documentation is important for your project maintainers (and for your future self!). A very nice documentation generator is [Docz](https://www.docz.site). It supports `mdx` files, which is short for Markdown with JSX. That means you can render React components in these special Markdown files. It can generate Prop tables and even provide a coding playground for your components.
```

Comment:
Minor nitpick, feel free to ignore :) this explains some of the things that Docz does, but it doesn't explain _why_ I should use Docz. Maybe this could say something like _Docz allows you to easily write interactive documentation for your React components._ and then talk about mdx, prop tables etc in the next paragraph?

Reply:
I gave that a try. :) Let me know your thoughts about the result!